.github/scripts/pr-find-issues.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash -e
+
+PR="$1"
+REPO="$2"
+
+if [ "$REPO" == "" ]; then
+  REPO="keycloak/keycloak"
+fi
+
+function parse_issues() {
+    echo "$1" | grep -i -P -o "(close|closes|closed|resolve|resolves|resolved) #[[:digit:]]*" | cut -d '#' -f 2 | sort -n
```

Comment:
The quotes around `$1` can be removed, and the string passed to cut should probably use double quotes like all the others.
```suggestion
    echo $1 | grep -i -P -o "(close|closes|closed|resolve|resolves|resolved) #[[:digit:]]*" | cut -d "#" -f 2 | sort -n
```

Reply:
Makes no difference, and what's there works just fine